PowerUp/app/src/main/java/powerup/systers/com/vocab_match_game/VocabMatchGameActivity.java

```diff
@@ -28,6 +28,9 @@
 import powerup.systers.com.sink_to_swim_game.SinkToSwimGame;
 import powerup.systers.com.datamodel.SessionHistory;
 
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.TRUE;
```

Comment:
No need of using static variables using import statements. You can directly use True and False.

Reply:
Got it! Will do that.